test/test_t2star.py

```diff
@@ -259,6 +258,57 @@ def test_t2star_parallel(self):
                 "Result quality bad for experiment on qubit " + str(i),
             )
 
+    def test_t2star_concat_2_experiments(self):
+        """
+        Concatenate the data from 2 separate experiments
+        """
+        unit = "s"
+        estimated_t2star = 30
+        estimated_freq = 0.7
+        # First experiment
+        qubit = 0
+        delays1 = list(range(1, 60, 2))
+
+        exp1 = T2StarExperiment(qubit, delays1, unit=unit)
+        default_p0 = {
+            "A": 0.5,
+            "t2star": estimated_t2star,
+            "f": estimated_freq,
+            "phi": 0,
+            "B": 0.5,
+        }
+        exp1.set_analysis_options(user_p0=default_p0, shots=2000)
+        backend = T2starBackend(
+            p0={
+                "a_guess": [0.5],
+                "t2star": [estimated_t2star],
+                "f_guess": [estimated_freq],
+                "phi_guess": [0.0],
+                "b_guess": [0.5],
+            },
+            initial_prob_plus=[0.0],
+            readout0to1=[0.02],
+            readout1to0=[0.02],
+            dt_factor=1,
+        )
+
+        # run circuits
+        expdata1 = exp1.run(backend=backend)
+
+        # second experiment
+        delays2 = list(range(2, 65, 2))
+        exp2 = T2StarExperiment(qubit, delays2, unit=unit)
+        exp2.set_analysis_options(user_p0=default_p0, shots=2000)
+        expdata2 = exp2.run(
+            backend=backend,
+            experiment_data=expdata1,
+        )
+        result = expdata2.analysis_result(0)
```

Comment:
I think `analysis_result(0)` is just the result of the first execution only. If you want the result of the combined two executions then refer to `analysis_result(1)`. Maybe verify that stderr is smaller in the second result?

Reply:
You are right (again). Good idea to check the stderr. Did both.